azurepipelines/build/native/adu-windows-msvc2022-amd64-build.yml

```diff
@@ -25,76 +25,104 @@ trigger:
+            displayName: "Build ADU Agent - Windows Server 2022 - MSVC 2022 (amd64)"
+            timeoutInMinutes: 60
+            cancelTimeoutInMinutes: 60
+            pool:
+              name: 1es_hosted_pool_windows
+              os: windows
+            steps:
+              - task: PowerShell@2
+                displayName: "Build Client + UTs - amd64_debug"
+                inputs:
+                  targetType: "filePath"
+                  filePath: $(Build.SourcesDirectory)\scripts\build.ps1
+                  arguments: >
                     -Type Debug
                     -Clean
                     -BuildUnitTests
                     -NoPrompt
 
-          - task: PowerShell@2
-            displayName: "Run Unit Tests - amd64_debug"
-            inputs:
-                targetType: inline
-                script: $(Build.SourcesDirectory)\scripts\run_uts.ps1 -Pipeline
+              - task: PowerShell@2
+                displayName: "Run Unit Tests - amd64_debug"
+                inputs:
+                  targetType: inline
+                  script: $(Build.SourcesDirectory)\scripts\run_uts.ps1 -Pipeline
 
-          - task: PublishTestResults@2
-            inputs:
-                testResultsFormat: 'JUnit'
-                testResultsFiles: '**/TEST-*.xml'
-                failTaskOnFailedTests: true
+              - task: PublishTestResults@2
+                inputs:
+                  testResultsFormat: 'JUnit'
+                  testResultsFiles: '**/TEST-*.xml'
+                  failTaskOnFailedTests: true
 
-          - task: PublishPipelineArtifact@1
-            inputs:
-              targetPath: 'out/Debug/bin/Debug'
-              artifact: 'bin-amd64_debug'
-              publishLocation: 'pipeline'
+              - task: 1ES.PublishPipelineArtifact@1
+                inputs:
+                  targetPath: 'out/Debug/bin/Debug'
+                  artifact: 'bin-amd64_debug'
+                  publishLocation: 'pipeline'
 
-          - task: PublishPipelineArtifact@1
-            inputs:
-              targetPath: 'out/Debug/lib/Debug'
-              artifact: 'lib-Debug-amd64'
-              publishLocation: 'pipeline'
+              - task: 1ES.PublishPipelineArtifact@1
+                inputs:
+                  targetPath: 'out/Debug/lib/Debug'
+                  artifact: 'lib-Debug-amd64'
+                  publishLocation: 'pipeline'
 
-    - job: BuildAduAgent_WindowsServer2022_msvc2022_arm64
-      displayName: "Build ADU Agent - Windows Server 2022 - MSVC 2022 (arm64)"
-      timeoutInMinutes: 60
-      cancelTimeoutInMinutes: 60
-      steps:
-          - task: PowerShell@2
-            displayName: "Build Client + UTs - arm64_release"
-            inputs:
-                targetType: "filePath"
-                filePath: $(Build.SourcesDirectory)\scripts\build.ps1
-                arguments: >
+          - job: BuildAduAgent_WindowsServer2022_msvc2022_arm64
```

Comment:
amd64?

Reply:
looks intentional. At some later point, would it be more clear to rename the yaml file?

Reply:
Also added to the work of yaml file cleanup